iptables/iptables.go

```diff
@@ -73,6 +73,7 @@ type IPTables struct {
 	v2             int
 	v3             int
 	mode           string // the underlying iptables operating mode, e.g. nf_tables
+	timeout        int    // time to wait for the iptables lock, default is 0, that means wait forever
```

Comment:
this is the only mutable field, right?
previously this struct looks like it was probably thread safe, having an unsynchronized getter/setter on this makes it not.

Reply:
yeah, you are right, better doing it in the constructor